dbmongo/js/.nycrc

```diff
@@ -8,6 +8,7 @@
     "**/*.js"
   ],
   "exclude": [
+    "**/*.d.ts",
```

Comment:
...sinon, on obtient l'erreur `Unable to require file: globals.d.ts` quand la CI exécute `nyc npm test`